pytorch_lightning/loggers/neptune.py

```diff
@@ -226,6 +230,7 @@ def __init__(
             name: Optional[str] = None,
             run: Optional[str] = None,
             prefix: str = '',
+            base_namespace: str = None,
```

Comment:
Set the default value to an empty string `''` as opposed to `None`, this will simplify the rest of the code.